src/js/utils/global-loader.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * An AMD loader that installs modules into the global edx namespace.
+ */
+(function() {
+    'use strict';
+
+    window.edx = window.edx || {};
+
+    window.edx.GlobalLoader = (function() {
+        var registeredModules = {};
+
+        // Register standard libraries
+        registeredModules.jquery = $;
```

Comment:
It's to make it so that HtmlUtils can load its dependencies:

https://github.com/edx/edx-ui-toolkit/blob/andya/minor-api-fixes/src/js/utils/html-utils.js#L9